python/cuml/cuml/experimental/fil/fil.py

```diff
@@ -151,13 +153,7 @@ def predict_proba(
             any power of 2, but little benefit is expected above a chunk size
             of 512.
         """
-        results = super().predict_proba(X, preds=preds, chunk_size=chunk_size)
-        if len(results.shape) == 2 and results.shape[-1] == 1:
-            results = results.to_output("array").flatten()
-            results = GlobalSettings().xpy.stack(
-                [1 - results, results], axis=1
-            )
-        return results
+        return super().predict_proba(X, preds=preds, chunk_size=chunk_size)
```

Comment:
It makes sense to keep the constructor to emit the `FutureWarning`, but do we still have to override `.predict()` and `.predict_proba()` ? The API between the two different namespaces `cuml.fil` and `cuml.experimental.fil` is the same, is it not?

Reply:
You're right. Let's remove it